src/shared/actions/schemaThunks.ts

```diff
@@ -47,25 +46,78 @@ export const fetchSchemaForBucket = async (
 
   const text = `from(bucket: "${bucket.name}")
   |> range(${range})
-  |> first()`
+  |> first()
+  |> drop(columns: ["_value"])
+  |> group()`
+
+  let ni, no
+  const filtered = [
+    /^_start$/,
+    /^_stop$/,
+    /^_time$/,
+    /^_value/,
+    /^_measurement$/,
+    /^_field$/,
+    /^table$/,
+    /^result$/,
+  ]
+  const res = await runQuery(orgID, text).promise.then(function generateSchema(
+    raw
+  ) {
+    if (raw.type !== 'SUCCESS') {
+      throw new Error(raw.message)
+    }
+
+    const out = fromFlux(raw.csv).table as any
+    const len = out.length
+    const measurements = out.columns._measurement.data
+    const fields = out.columns._field.data
+    const columns = out.columnKeys.filter(key => {
+      return filtered.reduce((acc, curr) => {
+        return acc && !curr.test(key)
+      }, true)
+    })
+    const colLen = columns.length
+    const schema = {} as any
```

Comment:
sorry for all of the anys... needed to be done using the same memory space for large datasets, so i couldn't just copy between types